plugins/algoltek-usb/meson.build

```diff
@@ -0,0 +1,18 @@
+if gusb.found()
```

Comment:
This strengthens my point you don't need a meson option. It's not even used here 😀

Reply:
So I removed the  line "If gusb.found() ","end if". Is that correct?